### PR TITLE
Updated the Appium Java READ.ME File

### PR DESCRIPTION
**Pull request by Devansh013**

### CODE REVIEW

**Review by shashankj-lt — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,71 +1,399 @@
...
+
+Here are sample JUnit automation scripts used to run on an Android app and iOS app. Ensure to update the `app_url` in the code scripts before running the tests.
+
+<Tabs className="docs__val">
+<TabItem value="android" label="Android" default>
+
+```java title="vanilla_android.java"
+import io.appium.java_client.AppiumDriver;
+import io.appium.java_client.MobileBy;
+import io.appium.java_client.MobileElement;
```
Comment (codeSolicitor) [inline README.md:115]: why this code is in read me file

**Review by codeSolicitor — CHANGES_REQUESTED** [review]
requested changes

File: README.md (hunk 1 above)
Comment (Devansh013) [reply README.md:115]: We have beautified the repos for all frameworks and thus creating an ecosystem where we aim to increase the visibility of LT on Github

Comment (Devansh013) [reply README.md:115]: Please merge these changes as soon as possible.

Comment (codeSolicitor) [reply README.md:115]: @Devansh013 this appium code is already in the java file why do you want to put it in read me file

**Review by shashankj-lt — DISMISSED** [review]

**Review by codeSolicitor — APPROVED** [review]
approved

**Pull request merged**